src/data/services/LmsApiService.js

```diff
@@ -384,6 +385,57 @@ class LmsApiService {
     const url = `${LmsApiService.baseUrl}/enterprise/api/v1/analytics-summary/${enterpriseUUID}`;
     return LmsApiService.apiClient().post(url, formData);
   }
+
+  static updateUserActiveEnterprise = (enterpriseId) => {
+    const url = `${configuration.LMS_BASE_URL}/enterprise/select/active/`;
+    const formData = new FormData();
+    formData.append('enterprise', enterpriseId);
+
+    return LmsApiService.apiClient().post(
+      url,
+      formData,
+    );
+  };
+
+  static loginRefresh = async () => {
+    const loginRefreshUrl = `${configuration.LMS_BASE_URL}/login_refresh/`;
+
+    try {
+      return LmsApiService.apiClient().post(loginRefreshUrl);
+    } catch (error) {
+      const isUserUnauthenticated = error.response?.status === 401;
+      if (isUserUnauthenticated) {
+        // Clean up the cookie if it exists to eliminate any situation
+        // where the cookie is not expired but the jwt is expired.
+        const cookies = new Cookies();
+        cookies.remove(configuration.ACCESS_TOKEN_COOKIE_NAME);
+      }
+      return Promise.resolve();
+    }
+  };
+
+  static fetchEnterpriseLearnerData(options) {
+    const enterpriseLearnerUrl = `${configuration.LMS_BASE_URL}/enterprise/api/v1/enterprise-learner/`;
+    const queryParams = new URLSearchParams({
+      ...options,
+      page: 1,
+    });
+    const url = `${enterpriseLearnerUrl}?${queryParams.toString()}`;
+    return LmsApiService.apiClient().get(url);
+  }
+
+  static getActiveLinkedEnterprise(username) {
+    return this.fetchEnterpriseLearnerData({ username }).then((response) => {
+      const { data } = response;
+      const results = data?.results;
+      for (let i = 0; i < results.length; i++) {
+        if (results[i].active) {
+          return results[i].uuid;
+        }
+      }
+      return '';
```

Comment:
If there is no active enterprise customer (super edge case, one should always exist), likely should return `undefined` or `null` instead of empty string.